exporter/neevafileexporter/README.md

```diff
@@ -0,0 +1,30 @@
+# Neeva File Exporter
+
+This is a file exporter for traces, which writes JSON-encoded gzipped files, and supports writing to S3. The specific JSON format was made to be backwards-compatible with other existing tooling. The file names that are written to are rotated based on time or (uncompressed) byte size.
```

Comment:
Any pointers to the "specific JSON format" and what it's "backward compatible" with?

Reply:
I kinda kept this vague because this is a fork, thus must be public, thus everything in this repo (including our commentary here) is public.